### PR TITLE
Make nteract root buildable on Windows

### LINKED ISSUE #3048: `npm install` from root does not work on Windows machines.

### PR DESCRIPTION
**Pull request by jdfreder**
This doesn't include any additional work that may be required to run
nteract applications/.

closes #3048

### CODE REVIEW

**Comment by jdfreder** [comment]
My atom has strip whitespaces enabled- is that okay @rgbkrk ?

**Comment by rgbkrk** [comment]
> My atom has strip whitespaces enabled- is that okay @rgbkrk ?

Yeah prettier usually normalizes things for us.

**Comment by rgbkrk** [comment]
Thanks for going after making dev work on Windows again!

**Comment by rgbkrk** [comment]
It looks like the style building stuff is failing in the prepare step on CI. Can you check that out? I can't tell what the issue is at first glance.

**Comment by jdfreder** [comment]
Yeah, looks like cp-cli wont copy to directories that already exist. I'm trying cpy-cli now (cpy based).

**Comment by jdfreder** [comment]
> Thanks for going after making dev work on Windows again!

It's a low hanging fruit - and it will allow me to participate in the sprints from my gaming rig :)

**Comment by jdfreder** [comment]
> It looks like the style building stuff is failing in the prepare step on CI. Can you check that out? I can't tell what the issue is at first glance.

Fixed

**Comment by alexandercbooth** [comment]
>It's a low hanging fruit - and it will allow me to participate in the sprints from my gaming rig :)

I do not know about everybody else, but I do not have access to a pc, so this is awesome! Windows support is vital. Thanks @jdfreder!

File: README.md [hunk 1]
```diff
@@ -96,6 +96,22 @@ When changes are made, they can then be pulled from the master branch:
...
+following from a Windows PowerShell launched as an Adminstrator:
+
+```
+npm install -g --production windows-build-tools
+```
+
+Additionally, you'll need to install GTK 2 for node-canvas. Follow [step 2 of
+this instruction
+set](https://github.com/Automattic/node-canvas/wiki/Installation---Windows#2-installing-gtk-2)
+to do this.
```
Comment (rgbkrk) [inline README.md:113]: And one day we will rid ourselves of this dependency. 😈

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
     "pack": "lerna run pack --scope nteract --stream",
     "dist": "lerna run dist --scope nteract --stream",
     "dist:all": "lerna run dist:all --scope nteract --stream",
-    "clean": "lerna clean --yes && rm -rf node_modules"
+    "clean": "lerna clean --yes && rimraf node_modules"
```
Comment (rgbkrk) [inline package.json:67]: Smart.

File: package.json [hunk 3]
```diff
@@ -210,7 +210,7 @@
     "util": "^0.10.3",
     "uuid": "^3.1.0",
     "webfontloader": "^1.6.28",
-    "webpack": "^4.0.1",
+    "webpack": "^4.14.0",
```
Comment (rgbkrk) [inline package.json:213]: 👍

File: package.json [hunk 4]
```diff
@@ -221,5 +221,9 @@
   },
   "engines": {
     "node": ">=8.9.0"
+  },
+  "dependencies": {
+    "cpy-cli": "^2.0.0",
+    "electron-prebuilt": "^1.4.13"
```
Comment (rgbkrk) [inline package.json:227]: This one [shouldn't be needed any more as electron ships its own prebuilt version now (electron prebuilt is deprecated)](https://github.com/electron-userland/electron-prebuilt).

File: packages/transform-dataresource/package.json [hunk 5]
```diff
@@ -10,7 +10,7 @@
     "build": "npm run build:clean && npm run build:lib",
     "build:clean": "rimraf lib",
     "build:flow": "flow-copy-source -v -i '**/__tests__/**' src lib",
-    "build:lib": "NODE_ENV=production babel -d lib src --ignore '**/__tests__/**'",
+    "build:lib": "cross-env NODE_ENV=production babel -d lib src --ignore '**/__tests__/**'",
```
Comment (rgbkrk) [inline packages/transform-dataresource/package.json:13]: Nice, thank you.

**Review by rgbkrk — APPROVED** [review]
Thank you @jdfreder !

**Comment by rgbkrk** [comment]
I'm going to go ahead and merge this since it passes, then I'm going to remove the extraneous dep.

**Comment by rgbkrk** [comment]
🎉 🖥

**Pull request merged**